### ISSUE TITLE
Rejigger `stat`/`maxstat` output

### ISSUE THREAD

**Issue by prophnoran** [prophecymud/ideas#118]
It's quite confusing to read. For example Noran's right now (yes yes I should probably fix wisdom at least) is:
```
 stat
-Str: 13[17] Int: 17[25] Wis: 13[19] Dex: 12[21] Con: 14[23]

 max 
-Str: 18[22] Int: 21[25] Wis: 18[22] Dex: 17[21] Con: 19[23]
```

How about just something like

```
stat
Without equipment (current/max)
Str: 13/18 Int: 17/21 Wis: 13/18 Dex: 12/17 Con: 14/19
With equipment (current/max)
Str: 17/22 Int: 25/25 Wis: 19/22 Dex: 21/21 Con: 23/23
```
I remember this being a bit annoying for beginners ("ok, run this command, look at those numbers, now run that command, look at those numbers") instead what we want is at a glance with one line.

Even better maybe would be two commands:

```
stat
Str: 17/22 Int: 25/25 Wis: 19/22 Dex: 21/21 Con: 23/23

rawstat
Str: 13/18 Int: 17/21 Wis: 13/18 Dex: 12/17 Con: 14/19
```
Because fairly rarely do we care about un-equipped stats, especially in a world where `train` doesn't train these stats.

**Comment by prophnoran** [comment]
(I guess `score` would also need updated)

**Comment by anhegofcherek** [comment]
Good call on the update, have just done that - it'll go live in the next deploy

Also, well done, looks like you've helped uncover an old bug!

Here's an excerpt from what `do_maxstats` looked like:

```
var MAX_STR_ITEM = ch.MaxStat(STAT_STR)
var MAX_STR_BASE = MAX_STR_ITEM - 4

ch.Chprintf("-`#Str``: `^%d``[`$%d``] etc.", MAX_STR_BASE, MAX_STR_ITEM, etc.)
```

Why 4? I asked that question myself. So I looked at what `do_train` did (before #117)

```
if ch.perm_stat[stat] >= get_max_train(ch, stat) {
    act("Your $T is already at maximum.", ch, nil, pOutput, TO_CHAR)
```

I can't easily paraphrase `get_max_train` and `MaxStats` but they're a bit more nuanced than just `-4`

Here's what I suspect happened:

* At some point it was literally just -4 and this was hardcoded in `do_maxstats`
* At some point `get_max_train` was introduced to add more complexity, along with whatever I ported to `MaxStats` (roughly speaking, if a function has C-style naming, it's a straight port from the C; if it's CamelCased I've refactored it to be more idiomatic Go)
* Nobody updated `do_maxstats`

So it's possible you'll see slightly different (but more correct) values when this goes live!
